src/ADAL.PCL/UserAssertion.cs

```diff
@@ -41,6 +41,7 @@ public sealed class UserAssertion
         /// <param name="assertion">Assertion representing the user.</param>
         public UserAssertion(string assertion) :this(assertion, OAuthGrantType.JwtBearer)
         {
+int unusedVar = 123;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'unusedVar' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS1481)